src/Routes.ts

```diff
@@ -0,0 +1,43 @@
+import express from "express";
+import { ReturnController } from "./controller/ReturnController";
+import { loggingHelper } from "./utils/LoggingHelper";
+import { EnvironmentVariables } from "./utils/EnvironmentVariables";
+
+export const register = ( router: express.Application ) => {
+	router.get("/resume", async (req, res) => {
+		try {
+			loggingHelper.info("Processing /resume");
+			const authorizeUrl = await ReturnController.getInstance().handleResumeReturnAuthUrl();
+			res.redirect(authorizeUrl);
```

Comment:
```suggestion
			const authorizeUrl = await ReturnController.getInstance().handleResumeReturnAuthUrl();
			loggingHelper.info("Successfully processed /resume");
			res.redirect(authorizeUrl);
```
is this worth it or is it overkill? i'm conscious from the logs without it we would never have positive confirmation of successful completion?